docs/spfx/web-parts/guidance/build-custom-property-pane-controls.md

```diff
@@ -309,21 +308,21 @@ export default class AsyncDropdown extends React.Component<IAsyncDropdownProps,
 }
 ```
 
-The **AsyncDropdown** class represents the React component used to render the asynchronous dropdown property pane control. When the component is loaded initially (the **componentDidMount** method) or its **disabled** or **stateKey** properties have changed, it will load the available options by calling the **loadOptions** method passed through the properties. Once loading options finished the component will update its state showing the available options. The dropdown itself is rendered using the [Office UI Fabric React dropdown component](http://dev.office.com/fabric#/components/dropdown). When the component is loading the available options it shows a spinner using the [Office UI Fabric React spinner component](http://dev.office.com/fabric#/components/spinner).
+The **AsyncDropdown** class represents the React component used to render the asynchronous dropdown property pane control. When the component first loads, the **componentDidMount** method, or its **disabled** or **stateKey** properties will change, and it will load the available options by calling the **loadOptions** method passed through the properties. Once the options are loaded the component updates its state showing the available options. The dropdown itself is rendered using the [Office UI Fabric React dropdown component](http://dev.office.com/fabric#/components/dropdown). When the component is loading the available options it displays a spinner using the [Office UI Fabric React spinner component](http://dev.office.com/fabric#/components/spinner).
```

Comment:
Without parenthesis surrounding _the componentDidMount method_ the sentence seems confusing. It was meant as a pointer to the right fragment in code but if it obstructs reading then we could leave it out.